.github/workflows/tests.yml

```diff
@@ -303,6 +303,8 @@ jobs:
           # `std` support still in progress. Can be moved up with the other
           # apple targets after https://github.com/rust-lang/rust/pull/103503
           aarch64-apple-tvos,
+          # `std` support being commited, see https://github.com/rust-lang/rust/pull/116527
+          i686-unknown-hurd-gnu,
```

Comment:
The PR was merged, so we can move it to the `std` targets section (though we may need wait for a new Nightly release).

Reply:
Indeed. The nightly release seems alright, tests now pass.